k8s/config/crd/bases/k8s.v6d.io_csidrivers.yaml

```diff
@@ -38,6 +38,9 @@ spec:
                       type: string
                   type: object
                 type: array
+              enableDebugLog:
+                default: false
+                type: boolean
```

Comment:
Can you let user set the log-level (as we previously discussed)?

Reply:
Okay, it looks like users only can set verbose or non-verbose. Do you think `EnableVerbose` is more meaningful?